POS/Troubleshooting/04_ConnectRetailSuite.md

```diff
@@ -3,6 +3,7 @@
 # Establish the RetailSuite connection
 
 #### Error description
+<!---Error-->
 
 <!---Ist das hier noch aktuell?-->
 The RetailSuite is not connected to *Venduo POS*. The *DATA TRANSFER RETAILSUITE ORDER MANAGEMENT* section is not displayed in the global settings.
```

Comment:
Vielleicht kurz erklären, was mit RetailSuite gemeint ist? Oder UCS Sync...? Das kommt nirgendwo in der Doku vor.